### DISCUSSION TITLE
FullDay events?

### DISCUSSION THREAD

**Discussion by apiening** [spatie/icalendar-generator#73] [Bugs]
The README.md says: 

It is possible to create an event that spans a full day:
```php
Event::create()
    ->fullDay()
    ...
```
When I try this, I get the following error:
```
Call to undefined method Spatie\IcalendarGenerator\Components\Calendar::fullDay()
```
Has the syntax changed or do I need to call the function differently?

**Comment by rubenvanassche** [comment]
Hi @apiening,

Just tried it myself and the method still exists and I don't get the error 🤔. Do you call any methods before `fullDay`? And what's the version of the package you're using?
